GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+- providing graphics and web design
+- programming
+- assisting with project infrastructure
+- writing documentation
+- fixing bugs
+- adding features
+
+Contributors engage with the project through the issue tracker and mailing
+list, or by writing or editing documentation. They submit changes to the
+project itself via patches, which will be considered for inclusion in the
+project by existing maintainers (see next section). The developer mailing list
+is the most appropriate place to ask for help when making that first
+contribution.
+
+As contributors gain experience and familiarity with the project, their profile
+within, and commitment to, the community will increase.
+A contributer who shows an above-average level of contribution to the project,
+particularly with respect to its strategic direction and long-term health, may
+be nominated to become a maintainer. This role is described below.
+
+## Maintainers 
+
+Maintainers are community members who have shown that they are committed to the
+continued development of the project through ongoing engagement with the
+community.
+
+Typically, a potential maintainer will need to show that they have an
+understanding of the project, its objectives and its strategy. They will also
+have provided valuable contributions to the project over a period of time.
+
+The project maintainers consists of those individuals identified as
+‘project owners’ on the development site. The maintainers have additional
+responsibilities over and above those of a contributor. These responsibilities
+ensure the smooth running of the project. Maintainers are expected to review
+code contributions, participate in strategic planning, approve changes to the
+governance model and manage the copyrights within the project outputs.
+
+Maintainers do not have significant authority over other members of the
+community, although they make decisions when community consensus cannot be
+reached. In addition, maintainers have access to the project’s private mailing
+list.  This list is used for sensitive issues, such as votes for new
+maintainers and legal matters that cannot be discussed in public. It is never
+used for project management or planning.
+
+Maintainer membership is by invitation from the existing members. A nomination
+will result in discussion and then a vote by the existing maintainers on the
+project’s private management list.
+Maintainer membership votes are subject to consensus approval of the current
+maintainer members.
+
+# Support
+
+All participants in the community are encouraged to provide support for new
+users within the project management infrastructure. This support is provided as
+a way of growing the community. Those seeking support should recognise that all
+support activity within the project is voluntary and is therefore provided as
+and when time allows. A user requiring guaranteed response times or results
+should therefore seek to purchase a support contract from a community member.
```

Comment:
There is no selling of support contracts from community members...

Reply:
What do you mean? Sure it is.. Red Hat is one. A non RH community member may want to provide it.
It happens in oVirt.

Reply:
I think it is good to specify this. Currently, there is no support contracts from community members (except Red Hat) but we are open to accept other "support vendors".

Reply:
> What do you mean? Sure it is.. Red Hat is one. A non RH community member may want to provide it.
> It happens in oVirt.

Red Hat is not a community member but an employer of community members. Also Red Hat does not offer a support contract for Nmstate but for RHEL.

Reply:
I do not understand where you are going with this.
Anyone can decide to support this project for free or for a paid price, there is nothing that limits that.

We are not stating here who this person or company is that provides such a support.

Reply:
IMHO the documentation should be concise and not include unnecessary sentences. Also, as long as nobody is offering this service, it is odd to advertise it. IMHO all content here should have a justification why it is important to have it. We would also not want to include unused code in the PRs for teh actual code.